validation/pom.xml

```diff
@@ -124,7 +124,7 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>com.coveo</groupId>
+                <groupId>com.spotify.fmt</groupId>
```

Comment:
nit: should we have made this change in different PR?

Reply:
The old one had conflicts during the build and it was outdated :(